sympy/stats/crv_types.py

```diff
@@ -1260,6 +1261,11 @@ def _cdf(self, x):
         return Piecewise((uppergamma(a,b/x)/gamma(a), x > 0),
                         (S.Zero, True))
 
+    def sample(self):
+        scipy = import_module('scipy')
+        if scipy:
```

Comment:
And if not, should an error be raised? Like `NotImplementedError('Sampling the inverse Gamma distribution requires SciPy')`? As is, the method returns None.